extensions/ql-vscode/src/view/model-editor/ModeledMethodDataGrid.tsx

```diff
@@ -51,8 +55,7 @@ export const ModeledMethodDataGrid = ({
         modeledMethods,
         methodIsUnsaved,
       );
-
-      if (methodCanBeModeled || !hideModeledMethods) {
+      if (shouldShowMethod(methodCanBeModeled, hideModeledMethods)) {
```

Comment:
The code here and in `MethodsUsageDataProvider` is very similar but not identical, so I didn't think it was sensible to combine `canMethodBeModeled` and `shouldShowMethod` into one shared method. Here we need both values while in the usages panel we only need to know whether to show it or not, and the method modelability it just an intermediate value.